main.js

```diff
@@ -1,63 +1,101 @@
-const sortSelect = document.getElementById('sort-select');
-sortSelect.addEventListener('change', submitNewSearch);
-
-const submitButton = document.getElementById('submit');
-submitButton.addEventListener('click', event => {
-  sortSelect.value = 'relevance';
-  submitNewSearch(event);
-});
-
-const toggleFiltersButton = document.getElementById('toggle-filters');
-toggleFiltersButton.addEventListener('click', handleToggleFiltersClick);
-
-const previousPageButton = document.getElementById('prev-page-btn');
-previousPageButton.addEventListener('click', handlePaginationClick);
-
-const nextPageButton = document.getElementById('next-page-btn');
-nextPageButton.addEventListener('click', handlePaginationClick);
-
 let articles;
-let totalHits;
 let resultsPage;
 
-function displayArticles() {
-  const searchResults = document.querySelector('.search-results');
+const sortSelect = document.getElementById('sort-by-select');
+const submitButton = document.getElementById('submit');
+const filtersButton = document.getElementById('filters-button');
+const previousPageButton = document.getElementById('previous-page-button');
+const nextPageButton = document.getElementById('next-page-button');
```

Comment:
which elements to you query up here and which ones do you query below? is there a rule for this?

Reply:
Any element that needs to be used by more than one function I've bound to a global variable. All other elements are bound locally. There is one exception: `submitButton`. It seems to have "snuck" into the global variables and should actually be declared locally within `bindEvents()` because that is the only place where it is accessed, so I'm going to relocate it.